python/rpdk/java/templates/init/guided_aws/StubCreateHandler.java

```diff
@@ -0,0 +1,142 @@
+package {{ package_name }};
+
+// TODO: replace all usage of SdkClient with your service client type, e.g; YourServiceAsyncClient
+//import com.amzn.my.resource.ClientBuilder.SdkClient;
```

Comment:
Should the import line be something like;

`import software.amazon.awssdk.services.yourservice.YourServiceAsyncClient;`